.github/workflows/website.yml

```
@@ -0,0 +1,77 @@
+    - name: Update website
+      run: |
+        cd target/threeten.github.io
+        git status
+  
+        rm -rf threeten-extra/
+        cp -R ../site threeten-extra/
+  
+        git add -A
+        git status
+        git commit --message "Update threeten-extra from CI: $GITHUB_RUN_ID"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)